docs/tutorial.md

```diff
@@ -41,15 +47,17 @@ $ $GOPATH/bin/nr-integrations-builder --help
 
 This tutorial assumes that your `$GOBIN` or `$GOPATH/bin` has been added to your `$PATH` environment variable.
 
-**Step 2: Check govendor tool**
+**Step 3: Check govendor tool**
 
 Before initializing the integration with `nr-integrations-builder` you have to check that the `govendor` tool (used for managing dependencies) is successfully installed. Run the following command:
 ```bash
 $ govendor
 ```
 You should receive the description about the `govendor` tool with the list of accepted commands. More information about the usage can be found in the [README.md](https://github.com/kardianos/govendor/blob/master/README.md).
 
-**Step 3: Initialize the integration**
+**Step 4: Initialize the integration**
+
+Create and `cd` to the `$GOPATH/company-name-integrations` directory. This is the directory where your integrations will be stored.
```

Comment:
Probably this `$GOPATH/company-name-integrations` should be the path that you created above: `$GOPATH/src/myorg-integrations/`, right?